<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="game">
      <attribute name="title">
       <string>Game</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Rows: </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Columns: </string>
       </property>
      </widget>
      <widget class="QSpinBox" name="rowsSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="columnSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Chests:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="chestsSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Rocks:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="rocksSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="player1">
      <attribute name="title">
       <string>Player 1</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="player1NameLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Player1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Character Breed:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="player1BreedBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Rogue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Barbarian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Troll</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UrukHai</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="player2">
      <attribute name="title">
       <string>Player 2</string>
      </attribute>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="player2NameLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Player2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Character Breed:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="player2BreedBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Rogue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Barbarian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Troll</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UrukHai</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
